docs/instructions.md

```diff
@@ -0,0 +1,29 @@
+# Configuration
+
+ItchySats on the Embassy requires a fully synced Electrum (electrs) node as a source for blockchain data. This requirement will be automatically enforced by EmbassyOS.
+The password can be found in the `Properties` section.
+
+# ItchySats
+
+ItchySats enables peer-2-peer CFD trading on Bitcoin using DLCs (discreet log contracts). No account needed, no trusted third-party - just you and your keys.
+
+This is beta software. We test it on test- and mainnet, but there are no guarantees that it will always behave as expected.
```

Comment:
🙃 

```suggestion
This is beta software. We test it on testnet and mainnet, but there are no guarantees that it will always behave as expected.
```